### PR TITLE
Show error Missing ssh private key

### PR DESCRIPTION
**Pull request by jeanp413**
Shows an error message telling the user to generate a SSH private key if it's missing

How to test:
1. remove any ssh private key
2. try to connect to a workspace
3. a error message should appear telling you to generate a SSH private key

### CODE REVIEW

File: extensions/gitpod/src/remoteConnector.ts [hunk 1]
```diff
@@ -458,11 +462,13 @@ export default class RemoteConnector extends Disposable {
...
+				authHandler(methodsLeft, _partialSuccess, _callback) {
+					if (methodsLeft !== null) {
+						return false;
+					}
 					return {
-						type: 'password',
-						username: workspaceId,
-						password: ownerToken,
+						type: 'none',
+						username: sshDestInfo.user,
```
Comment (jeanp413) [inline extensions/gitpod/src/remoteConnector.ts:471]: Doing more tests seems it's not that easy, using `type: 'none'` and `username` still fails to connect so I guess we need to check for private key files manually :disappointed: and just use `type: 'password'`
